lib/commands/new.js

```diff
@@ -64,6 +71,13 @@ module.exports = Command.extend({
       return Promise.reject(new SilentError(message));
     }
 
+    let langDef = this.availableOptions.find((opt) => opt.name === 'lang').default;
```

Comment:
AFAICT there is no default anymore, we should be able to remove this.

Reply:
Removed when consolidating `lang`-related logic into `init`  in f95d3a9  👍